lib/udap_security_test_kit/authorization_code_redirect_test.rb

```diff
@@ -20,7 +20,7 @@ class AuthorizationCodeRedirectTest < Inferno::Test
 
     receives_request :redirect
 
-    config options: { redirect_uri: "#{Inferno::Application['base_url']}/custom/udap_security_test_kit/redirect" }
+    config options: { redirect_uri: "#{Inferno::Application['base_url']}/custom/udap_security/redirect" }
```

Comment:
Could you extract this url somewhere so that we don't have to define it in multiple locations?

Reply:
Done, extracted it to its own constant file (since it needs to be accessed by the test suite and the software statement builder module) and configured the redirect_uri option for the entire suite instead of per test.